docs/ref/django-admin.txt

```diff
@@ -240,6 +240,15 @@ are prefixed by ``"###"``.
 The settings module to compare the current settings against. Leave empty to
 compare against Django's default settings.
 
+.. django-admin-option:: --output
+
+.. versionadded:: 2.0
+
+Specifies the output format. Available values are ``hash`` and
+``unified``. Default is ``hash``. The ``hash`` mode displays the output similar
+to ``"postconf -n"`` while the ``unified`` mode displays the output similar to
```

Comment:
Removed the "postconf -n" reference. Had used it to keep the original author's intent. The `hash` shows the output exactly like what Django used to do until now. So I've mentioned that.

Added notes in the `docs/releases/2.0.txt as well`.